persistence/trees/trees.go

```diff
@@ -72,26 +72,27 @@ type stateTree struct {
 	nodeStore kvstore.KVStore
 }
 
-var _ modules.TreeStoreModule = &treeStore{}
+var _ modules.TreeStoreModule = &TreeStore{}
 
-// treeStore stores a set of merkle trees that
+// TreeStore stores a set of merkle trees that
 // it manages. It fulfills the modules.TreeStore interface.
 // * It is responsible for atomic commit or rollback behavior
 // of the underlying trees by utilizing the lazy loading
 // functionality provided by the underlying smt library.
-type treeStore struct {
+type TreeStore struct {
 	base_modules.IntegrableModule
 
-	logger       *modules.Logger
-	treeStoreDir string
+	logger *modules.Logger
+
+	TreeStoreDir string
 	rootTree     *stateTree
 	merkleTrees  map[string]*stateTree
 }
 
-// GetTree returns the name, root hash, and nodeStore for the matching tree tree
-// stored in the TreeStore. This enables the caller to import the smt and not
-// change the one stored
-func (t *treeStore) GetTree(name string) ([]byte, kvstore.KVStore) {
+// GetTree returns the root hash and nodeStore for the matching tree
+// stored in the TreeStore. This enables the caller to import the smt
+// and not change the one stored.
```

Comment:
NIT 
```suggestion
// and not change the one stored, unless they call `Commit()` to write to the nodestore
```

Reply:
I reformatted this comment to capture this change 👍